### PR TITLE
Add CRS.distance docstring

### PR DESCRIPTION
**Pull request by nathancahill**
Undocumented at the moment.

### CODE REVIEW

File: src/geo/crs/CRS.js [hunk 1]
```diff
@@ -73,6 +73,9 @@ L.CRS = {
 		return L.bounds(min, max);
 	},
 
+	// @method distance(latlng1: LatLng, latlng1: LatLng): Number
+	// Returns the distance between two geographical coordinates.
```
Comment (perliedman) [inline src/geo/crs/CRS.js]: Would be great to add that the returned value is in meters!

Comment (nathancahill) [inline src/geo/crs/CRS.js]: distance is not necessarily in meters, only for L.CRS.Earth and it's inheritants. CRS.Simple is just euclidean distance.

Comment (nathancahill) [inline src/geo/crs/CRS.js]: Added notes about distance() return values to each of those descriptions.

Comment (perliedman) [inline src/geo/crs/CRS.js]: Right! I should have known that! :smile:

**Pull request merged**